### ISSUE TITLE
Maybe I know why there are so many boxes

### ISSUE THREAD

**Issue by LiaoLIDIP** [qijiezhao/M2Det#83]
Thanks for you great work! 
The reason why so many bboxes is that the pre-train weight is not completed, M2Det is so deep but do not use res block, But the pre-train weight is only bockbone net, when train this model with small batch size or big LR, gradient explosion will appear (you can see the train loss curve). 
I train vgg_512 on VOC with the author pre-train weight, and got good result (with 10% train dataset, mAP=0.8059) .
So, can you give more pre-train weights like vgg_320 resnet101, because I don't have many GPUs to train on original VOC or COCO. -.-
出现很多框的原因我实验得出是因为加载预训练模型的时候只加载了主干网络，而检测网络深度也是非常深的，由于网络没有残差结构，这样训练很容易出现梯度爆炸。
我在作者给出的vgg_512模型的基础上只改最后输出层（改成21类），在VOC0712上用10%的训练数据训练，测试集上mAP达到0.8059.
作者能不能给出vgg_320 resnet101 的模型训练结果呀？比较我只有一个GPU， 没法从头全部训练VOC 或者COCO。。。
多谢！

**Comment by Code-Fight** [comment]
@LiaoLIDIP 你好，你上面提到了修改作者的预训练模型的输出层， 这个怎么修改？谢谢

**Comment by arc144** [comment]
I've tried training with 4 T4 16GB GPUs and it also did not work. What worked for me was changing the classification loss for Focal loss. The training was much smoother and solved the excessive box problem.
